projects/plugins/jetpack/modules/sso/class.jetpack-sso-notices.php

```diff
@@ -4,202 +4,233 @@
 
 if ( ! class_exists( 'Jetpack_SSO_Notices' ) ) :
 
-/**
- * A collection of helper functions used in the SSO module.
- *
- * @since 4.4.0
- */
-class Jetpack_SSO_Notices {
 	/**
-	 * Error message displayed on the login form when two step is required and
-	 * the user's account on WordPress.com does not have two step enabled.
+	 * A collection of helper functions used in the SSO module.
 	 *
-	 * @since 2.7
-	 * @param string $message
-	 * @return string
-	 **/
-	public static function error_msg_enable_two_step( $message ) {
-		$error = sprintf(
-			wp_kses(
-				__(
-					'Two-Step Authentication is required to access this site. Please visit your <a href="%1$s" rel="noopener noreferrer" target="_blank">Security Settings</a> to configure <a href="%2$s" rel="noopener noreferrer" target="_blank">Two-step Authentication</a> for your account.',
-					'jetpack'
+	 * @since 4.4.0
+	 */
+	class Jetpack_SSO_Notices {
```

Comment:
Because of the `if` statement above, everything within this `if` block needed to be intended. That's the majority of the changes, but some doc updates since those lines were changed.